pkg/crypto/provider/vaultkeystore/hashicorp_vault_keystore_test.go

```diff
@@ -0,0 +1,231 @@
+package vaultkeystore_test
+
+import (
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/inklabs/rangedb/pkg/crypto"
+	"github.com/inklabs/rangedb/pkg/crypto/cryptotest"
+	"github.com/inklabs/rangedb/pkg/crypto/provider/vaultkeystore"
+)
+
+func TestHashicorpVault_VerifyEngineInterface(t *testing.T) {
+	config := getConfigFromEnvironment(t)
+	const iv = "1234567890123456"
```

Comment:
iv?

Reply:
initialization vector for AES encryption. I forgot to remove encryption from this keystore. Great catch @tomParty